scout/adapter/client.py

```diff
@@ -44,6 +43,8 @@ def get_connection(
 
     """
     authdb = authdb or mongodb
+    log_uri = uri
```

Comment:
I guess what he was trying to do is keep passwords out of the log. To me that makes a bit of sense. Perhaps just say connecting to `"mongod using config URI string"`, and reserve the full URI output for debug? That way, production logs would not contain passwords.

Reply:
I think I'll just remove it, it's not something that we see anyway. And if there are problems with the connection we would see in the `LOG.error(f"Database connection error:{err}")` line